### DISCUSSION TITLE
No Accuracy When Segmenting

### DISCUSSION THREAD

**Discussion by dpardi02** [Project-MONAI/MONAILabel#1551] [Q&A]
Hello,
I have been working with MonaiLabel for the segmentation of 3D volumes but have run into some issues when I go to train the model. Despite having 25% of my dataset labeled, there is no accuracy in the training. While the labels may not be perfect as this is a working dataset, I was expecting some type of accuracy to appear. I have had success with MonaiLabel for segmentation before so I am unsure what the issue might be in this case. I would be able to add any extra details that may be needed to help contextualize the issue but are there any common problems I might be able to check first? Thank you for any help you are able to provide.

**Comment by diazandr3s** [comment]
Hi @dpardi02,

Could you please expand on which model are you using? Which image modality and to segment what?
What do you mean by no accuracy in the training? Could you please share the logs you're seeing?
